LICENSE

```diff
@@ -19,3 +21,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+## About the Tellor Development Community
+
+The Tellor Development Community is the set of all contributors to the Tellor
+project. This includes all of the Tellor subprojects.
+
+The core team that coordinates development on GitHub can be found at
+https://tellor.io
+
+## Our Copyright Policy
```

Comment:
Let's just delete this part.... we don't want any contributor having ownership over their contributions.  Contribute at your own risk and donate it... there could be legal issues w/ this wording